File: tutorial/code/java/LED.java
import com.zilogic.zio.*; // <1>

class LED {
    public static void main(String args[]) 
	throws ProtocolException, InterruptedException {

	Agent agent = new Agent("/dev/ttyUSB0"); // <2>
	GPIO gpio = new GPIO(agent); // <3>
	int ledPin = 12; // <4>
	
	while (true) {
	    gpio.writePin(ledPin, 1); // <5>
	    Thread.sleep(1000);
	    
	    gpio.writePin(ledPin, 0); // <5>
	    Thread.sleep(1000);
	}
    }
}

File: tutorial/code/python/led.py
import zio # <1>
import time

agent = zio.Agent("/dev/ttyUSB0") # <2>
gpio = zio.GPIO(agent) # <3>
led_pin = 12 # <4>

while True:
    # Turn on the LED
    gpio.write_pin(led_pin, 1) # <5>
    time.sleep(1)

    # Turn off the LED
    gpio.write_pin(led_pin, 0) # <5>
    time.sleep(1)      
